CHANGELOG.asciidoc

```diff
@@ -3,6 +3,16 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-6.8.4]]
+=== Beats version 6.8.4
+https://github.com/elastic/beats/compare/v6.8.3...v6.8.4[View commits
```

Comment:
this is missing a closing `]`. There might be more.

Reply:
Good catch.  I've fixed the missing `]` in several spots.